### PR TITLE
[MTM-55154] crl proper error code fix

### PR DESCRIPTION
**Pull request by sguhc8y**
Fix for incorrect response code for CRL entry

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: CRL entry response status with correct error code
```
Comment (sarangovindarajan) [inline content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:3]: Rephrase: _Fixed the http response code for wrong CRL entries_

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-55154
+version: 10.20.294.0
+---
+Wrong error code being shown for a missing field in CRL entry payload is now fixed.
```
Comment (sarangovindarajan) [inline content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:17]: _Wrong http response code was sent when there are missing fields in CRL entry. This is fixed now._

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md (hunk 1 above)
Comment (kalavathig1) [reply content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:3]: I would suggest `Fixed the http response code for missing fields in CRL entries`

**Comment by kalavathig1** [comment]
Please don't forget to graft it to y2024 branch

**Review by kalavathig1 — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-55154
+version: 10.20.294.0
+---
+Wrong http response code was sent when there are missing fields in CRL entry. This is fixed now.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:17]:
```suggestion
A wrong HTTP response code was sent when fields were missing in the CRL entry. This behavior has been fixed now.
```

Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:17]: What is CRL? Please do not abbreviate without context.

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed the http response code for missing fields in CRL entries
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:3]:
```suggestion
title: Correct HTTP response code sent on missing fields in CRL entries
```

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md (hunk 3 above)
Comment (sguhc8y) [reply content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md]: Please check now

File: content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Correct HTTP response code sent on missing fields in certificate revocation list (CRL) entries
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-294-0-crl-proper-error-code.md:3]:
```suggestion
title: Correct HTTP response code sent on missing fields in CRL (Certificate Revocation List) entries
```

**Review by BeateRixen — APPROVED** [review]

**Review by pink-mongoose — APPROVED** [review]

**Pull request merged**